tools/gulptasks/unsorted/update-devDependencies.js

```diff
@@ -0,0 +1,30 @@
+/* eslint no-console: 0 */
+const gulp = require('gulp'),
+    path = require('path');
+/**
+ * Update the updateDevDependencies files to resolve vulnerabilities
+ *
+ * @return {Promise<Array<*>>}
+ *         Promises to keep
+ */
```

Comment:
This looks more like a code patch. The comment should be corrected and the task name should be something like `patch-ink-docstrap`